src/couch/src/couch_httpd.erl

```diff
@@ -18,7 +18,14 @@
 
 -export([header_value/2, header_value/3, qs_value/2, qs_value/3, qs/1, qs_json_value/3]).
 -export([path/1, absolute_uri/2, body_length/1]).
--export([verify_is_server_admin/1, unquote/1, quote/1, recv/2, recv_chunked/4, error_info/1]).
+-export([
+    verify_is_server_admin/1,
+    unquote/1,
+    quote/1,
+    recv/2,
+    recv_chunked/4,
+    error_info/1
+]).
```

Comment:
If this didn't change the content of the export list, let's keep the formatting as is.

Reply:
I believe that was the result of erlfmt when I had `unquote_path/1` being exported. I have changed it back.